src/catalog/src/remote/region_alive_keeper.rs

```diff
@@ -114,10 +129,11 @@ impl RegionAliveKeepers {
         keeper.deregister_region(region_ident.region_number).await
     }
 
-    pub async fn start(&self, heartbeat_interval_millis: u64) {
+    pub async fn start(&self) {
         for keeper in self.keepers.lock().await.values() {
-            keeper.start(heartbeat_interval_millis).await;
+            keeper.start().await;
         }
+        self.started.store(true, Ordering::Relaxed);
     }
```

Comment:
I guess we should execute `self.started.store(true, Ordering::Relaxed);` first?
```suggestion
    pub async fn start(&self) {
        self.started.store(true, Ordering::Relaxed);
        for keeper in self.keepers.lock().await.values() {
            keeper.start().await;
        }
    }
```

Reply:
The region alive keeper will be first inserted into `keepers`, then check start. See `register_table` method. If we set `started = true` first here, the keeper might be started twice.

Reply:
Is it possible that set `stated` later might cause some keepers don't start?